spec/policies/health_insurance_policy_spec.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe HealthInsurancePolicy do
+  let(:user) { create(:user) }
+  let(:health_insurance_user) { create(:health_insurance, user: user) }
+  let(:health_insurance_no_user) { create(:health_insurance) }
```

Comment:
Whenever using `entity_no_user`, it would be better `entity_withouth_user`

Reply:
Done! b0457a4adda1a2067db56f4d4a7937a82eb9be28